src/helpers/transactionPressHandler.ts

```diff
@@ -0,0 +1,143 @@
+import lang from 'i18n-js';
+import { getHumanReadableDate, hasAddableContact } from './transactions';
+import { isValidDomainFormat } from './validators';
+import { TransactionStatusTypes } from '@rainbow-me/entities';
+import TransactionActions from '@rainbow-me/helpers/transactionActions';
+import { useNavigation } from '@rainbow-me/navigation';
+import Routes from '@rainbow-me/routes';
+import { colors } from '@rainbow-me/styles';
+import {
+  abbreviations,
+  ethereumUtils,
+  showActionSheetWithOptions,
+} from '@rainbow-me/utils';
+
+const startCase = (string: string) =>
+  string.charAt(0).toUpperCase() + string.slice(1);
+
+export const transactionPressBuilder = ({
```

Comment:
If this is only used in a single file, do we need to have it extracted into the shared `/helpers` directory? Maybe we should consider patterns of co-locating large handlers like this alongside their components. This is something I'm thinking about 🤔

Reply:
Absolutely! Very unlikely that this extremely specific function will be useful anywhere else (and trying to make it work in another place would probably introduce a ton of bugs anyway).

I will move it.

Reply:
Hm, `src/helpers/transactionPressHandler.ts` is used in `buildTransactionsSectionsSelector.tsx` and they are both in `src/helpers`.

The second file has one and only usage in `useAccountTransactions.js`, but... It's in `src/hooks`. And that hook is used in 2 places. So it's hard to come up with a better place for it as it is now 😬.

Reply:
Oh my mistake! Fair enough, thanks for looking into it :)